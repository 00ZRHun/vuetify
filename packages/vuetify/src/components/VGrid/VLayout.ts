import factory from './grid'

export default factory('layout')
