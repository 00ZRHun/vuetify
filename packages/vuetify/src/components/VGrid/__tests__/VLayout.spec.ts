// Components
import VLayout from '../VLayout'

// Utilities
import {
  mount,
  MountOptions,
  Wrapper
} from '@vue/test-utils'

describe('VLayout.ts', () => {
  type Instance = InstanceType<typeof VLayout>
  let mountFunction: (options?: MountOptions<Instance>) => Wrapper<Instance>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VLayout, options)
  })

  it('should render', () => {
    const wrapper = mountFunction()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
