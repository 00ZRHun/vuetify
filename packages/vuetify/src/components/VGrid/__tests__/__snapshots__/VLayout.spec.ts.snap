// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VLayout.ts should render 1`] = `

<div class="layout">
</div>

`;
